tests/test_lsst.py

```diff
@@ -0,0 +1,928 @@
+from __future__ import with_statement
+import unittest
+import numpy as np
+import warnings
+import os
+import galsim
+from galsim.lsst import LsstCamera, LsstWCS, _nativeLonLatFromRaDec
```

Comment:
I think these should use the standard unittest skipping infrastructure. Set a flag as you have in the modification above and then use the `skipif` decorator for the tests.